13/main/index.html

```diff
@@ -0,0 +1,1480 @@
+                                <br />   </span>
+                                <span class="hidden" id="stomp_header">  Stomp the yard</span>
+                                <span class="hidden" id="stomp_tagline">  </span>
+                                <span class="hidden" id="stomp_image"> <img alt='image' src="images/inaugral.jpg" /> </span>
+
+
+
+                                <span class="event_name"> Antakshari </span>
+                                <span class="bottomSection hidden" id="antakshari_content">
+For people whose lives revolve around Indian music and can come up with a song for any situation or mood in an instant, THIS is the event to be a part of! Event will be on 22, 23, 24 of February on-campus. <br /><br/>
+                                <br />   </span>
+                                <span class="hidden" id="antakshari_header">  Antakshari </span>
+                                <span class="hidden" id="antakshari_tagline">  </span>
+                                <span class="hidden" id="antakshari_image"> <img alt='image' src="images/inaugral.jpg" /> </span>
+
+
+
+                                <span class="event_name"> Euphonica </span>
+                                <span class="bottomSection hidden" id="euphonica_content">
+There are people who have music imbibed in their every breath, thought and actions. We in IIIT believe that you are indeed one of those rare few blessed musicians who deserve to participate in this event. <br /><br /> Registration on-site (22, 23, 24 February)
+                                <br />   </span>
+                                <span class="hidden" id="euphonica_header">  Euphonica </span>
+                                <span class="hidden" id="euphonica_tagline">  </span>
+                                <span class="hidden" id="euphonica_image"> <img alt='image' src="images/inaugral.jpg" /> </span>
+
+
+
+                                <span class="event_name"> Matchitecture</span>
+                                <span class="bottomSection hidden" id="match_content" style="display:none">
+This fun and creative event involves building your dream house- with a twist. Participants can only use matchboxes.. and no adhesive!
+In one hour, you can relive your childhood of building sandcastles using matchsticks and boxes instead.
+Participants are provided with 50-60 matchboxes and nothing else to create an eclectic structure of their own imagination.
+Creations are judged based on the length, creativity and stability of the structure. Out of the box thinking is always appreciated.
+                                <br />   </span>
+                                <span class="hidden" id="match_header">  Matchitecture </span>
+                                <span class="hidden" id=
```

Comment:
Okay, this is really weird. In my emails and on the main PR page, the relevant lines shown are completely unrelated - which is why I didn't get what you were talking about.

Also, how about we leave this? :P I don't see much point of tracking these, and let the old guys see if they ever open GA? :P